### ISSUE TITLE
Presentation mode in a multi-display setup on Linux (KDE/Wayland)

### ISSUE THREAD

**Issue by b2cc** [ONLYOFFICE/DesktopEditors#1548]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package)

### Version information

8.0.1.31

### Expected Behavior

Possibility to select screen on which the presentation should be displayed for presentation mode and notes

### Actual Behavior

Presentation opens on "wrong" screen

### Reproduction Steps

 - Install a Debain-Linux distribution with the KDE Plasma 5 desktop and Wayland display server
 - Connect two more more displays to the computer
 - Install OnlyOffice 8.x
 - Run a presentation
 - No possibility to select screen for presentation, presentation starts on "wrong" screen

This is a follow-up issue to https://github.com/ONLYOFFICE/DesktopEditors/issues/285, see my comment there: https://github.com/ONLYOFFICE/DesktopEditors/issues/285#issuecomment-2036414774

---

### Additional information

At least under KDE/Wayland, this doesn't work as intended. I have no Xorg setup anymore to test if this is Wayland-only though.

E.g. I have a 3-monitor setup, with the left monitor being turned 90° (see below). The center monitor has the highest priority, followed by the right and left monitors. However, the presentation always starts on 90°-turned left monitor. I haven't found any way to force the presentation window to another screen. Even If I make the application start on the middle or right screen, the presentation always goes to the left screen.

Thinking of how many possible configurations exist out there I believe it would be best if we had the possibility to chose the screens for the presentation and the notes before starting the presentation (e.g. displaying a small window with the detected screen layout and being able to select the screens by ourselves).

![oo1](https://github.com/ONLYOFFICE/DesktopEditors/assets/24914972/d99f03aa-bcd1-47a8-8103-dd97580c49ac)

**Comment by jleivo** [comment]
I have similar issue.

I have extensive display setup on Windows. I have 7 displays. 
I cannot control the screen where the presentation slides appear. They always appear on the screen number 4.

The flow I start with is application is on screen 1. I choose the presenter view.
Presenter slides appear on screen 1 and presentation on screen 4. There is no way to move the presentation to any other screen.
![image](https://github.com/user-attachments/assets/237d1116-925b-4aef-8f2a-e048eb2073b3)

**Comment by davidgood** [comment]
I have the same issue. OnlyOffice recognizes one display, and I don't have any option to configure the slide show.  I connect to a projector via HDMI but OnlyOffice doesn't have any option to display on the projector and give me the presenter view on my laptop.

Fedora 41
Wayland
KDE Plasma 6.2.3
OO 8.2.1.38

**Comment by bhaubeck** [comment]
Same here. 

Fedora 41
Wayland
KDE Plasma 6.2.3
OO 8.2.1.38

I`d really like to chose the screen for the presentation or - at least - get a mode "Show presentation in a window" like Libre Office is offering, so you can move the window with presentation on the screen you would like to have it. 
ben

**Comment by dajomu1** [comment]
Since this issue has been present for several years I wonder if the developers are actually using their own software under Linux. Is this not one of the main things that should work? I am not a developer, but how hard is this? I see other open source software like openlp, libreoffice, etc that make this work.

**Comment by adiee5** [comment]
Can confirm, I had a problem displaying the presentation on wayland and had to switch to X11 in order to show the presentation (I use GNOME)

**Comment by dajomu1** [comment]
So, any action on this bug?

**Comment by WigF1** [comment]
Aaaaand still an issue. 

We still have no ability to select the actual monitor we would like to use for the presentation.
